core/Command/User/Add.php

```diff
@@ -82,12 +100,33 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 			return 1;
 		}
 
+		$password = '';
+		$sendPasswordEmail = false;
+
 		if ($input->getOption('password-from-env')) {
 			$password = getenv('OC_PASS');
+
 			if (!$password) {
 				$output->writeln('<error>--password-from-env given, but OC_PASS is empty!</error>');
 				return 1;
 			}
+		} elseif (\mb_strlen($input->getOption('email')) > 0) {
```

Comment:
![grafik](https://github.com/nextcloud/server/assets/213943/266a9a00-665f-48da-a7e3-b00278fe6610)

```suggestion
		} elseif ($input->getOption('email') !== '') {
```